### PR TITLE
Move cr3 extract ocr narrative script from airflow repo to vz-data

### PR DESCRIPTION
**Pull request by chiaberry**
## Associated issues

https://github.com/cityofaustin/atd-data-tech/issues/11731

The script used to live in [the airflow repo](https://github.com/cityofaustin/atd-airflow/blob/master/dags/python_scripts/cr3_extract_diagram_ocr_narrative.py). I have moved it and made it into a Docker image, copying the pattern Frank started with the other VZ ETLs. 

## Testing
**Steps to test:**

Either test via the airflow PR (https://github.com/cityofaustin/atd-airflow/pull/123) or pull `atddocker/atd-vz-cr3-extract:production` and run 
```
python ./cr3_extract_diagram/cr3_extract_diagram_ocr_narrative.py -v -d --update-narrative --update-timestamp --batch 100 --cr3-source atd-vision-zero-editor production/cris-cr3-files --save-diagram-s3 atd-vision-zero-website cr3_crash_diagrams/production
```

---
#### Ship list
- [ ] Code reviewed
- [ ] Product manager approved

### CODE REVIEW

File: .github/workflows/build_docker_images.yml [hunk 1]
```diff
@@ -65,3 +70,12 @@ jobs:
           context: atd-etl/cris_import
           push: true
           tags: atddocker/vz-cris-import:${{ github.ref == 'refs/heads/production' && 'production' || 'latest' }}
+
```
Comment (chiaberry) [inline .github/workflows/build_docker_images.yml:73]: fully copying and pasting Frank's magic!

File: atd-etl/cr3_extract_diagram/cr3_extract_diagram_ocr_narrative.py
Comment (chiaberry) [inline atd-etl/cr3_extract_diagram/cr3_extract_diagram_ocr_narrative.py:1]: I did not change this file at all, full copy and paste from https://github.com/cityofaustin/atd-airflow/blob/master/dags/python_scripts/cr3_extract_diagram_ocr_narrative.py

File: atd-etl/cr3_extract_diagram/cr3_extract_diagram_ocr_narrative.py [hunk 2]
```diff
@@ -0,0 +1,374 @@
...
+    if args.v:
+        print("Rendering PDF into images")
+    pages = []
+    try:
+        pages = convert_from_bytes(pdf["Body"].read(), 150)
+    except:
+        sys.stderr.write(
+            "Error: PDF Read for crash_id (" + str(crash["crash_id"]) + ") failed.\n"
+        )
+        continue
```
Comment (chiaberry) [inline atd-etl/cr3_extract_diagram/cr3_extract_diagram_ocr_narrative.py:240]: I am getting an error when I run this from the docker container, 
```
[2023-07-12, 14:34:58 CDT] {docker.py:391} INFO - Preparing to operate on crash_id: 19638112
Pulling CR3 PDF from S3
[2023-07-12, 14:34:58 CDT] {docker.py:391} INFO - Rendering PDF into images
[2023-07-12, 14:34:59 CDT] {docker.py:391} INFO - Error: PDF Read for crash_id (19638112) failed.```

still trying to figure out why

**Review by mddilley — APPROVED** [review]
I tested this in its Airflow companion PR, and everything worked! Thanks for closing out the Prefect v1 era! 🏁 🎉 🙌

File: atd-etl/cr3_extract_diagram/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+RUN apt-get install -y poppler-utils tesseract-ocr
+
+# Copy our own application
+WORKDIR /app
+COPY . /app/cr3_extract_diagram
+
+RUN chmod -R 755 /app/*
+
+# Install requirements
+RUN cd /app/cr3_extract_diagram && apt-get update && \
```
Comment (frankhereford) [inline atd-etl/cr3_extract_diagram/Dockerfile:14]: I think you can zap the `&& apt-get update` out of this command. There's no need to refresh the debian package manifests in the course of installing python requirements. It's also covered on line 3.

File: atd-etl/cr3_extract_diagram/env_template
Comment (frankhereford) [inline atd-etl/cr3_extract_diagram/env_template:1]: I think you can either remove this file or you could bring over the `docker-compose.yaml` file and set it up so you testing / execution can happen outside of the airflow context with something like `docker compose run <service>` or something like that. Do you have a `.env` file that you use?

**Review by frankhereford — APPROVED** [review]
This looks awesome Chia! I have a few observations, but nothing that should keep this from shipping!!

File: atd-etl/cr3_extract_diagram/env_template
Comment (chiaberry) [reply atd-etl/cr3_extract_diagram/env_template:1]: yeah, i used a local env file in order to test it locally. I included the env_template so if someone was going to run it locally, they had a set of what vars are needed in their .env file

File: atd-etl/cr3_extract_diagram/Dockerfile (hunk 3 above)
Comment (chiaberry) [reply atd-etl/cr3_extract_diagram/Dockerfile]: thank you!

**Pull request merged**